cub/device/device_histogram.cuh

```diff
@@ -178,9 +178,9 @@ struct DeviceHistogram
      * int      num_row_samples;    // e.g., 5
      * int      num_rows;           // e.g., 2;
      * size_t   row_stride_bytes;   // e.g., 7 * sizeof(float)
-     * float*   d_samples;          // e.g., [2.2, 6.0, 7.1, 2.9, 3.5,   -, -,
-     *                              //        0.3, 2.9, 2.0, 6.1, 999.5, -, -]
-     * int*     d_histogram;        // e.g., [ -, -, -, -, -, -, -, -]
```

Comment:
It seems that the same fix is required for two overloads of `HistogramRange`.

Reply:
Oops, I thought I'd checked the other examples. I'll update this before merging.